diff --git a/codebase/libpng16/configure.ac b/codebase/libpng16/configure.ac
index 17a6a2f..0d2a82e 100644
--- a/codebase/libpng16/configure.ac
+++ b/codebase/libpng16/configure.ac
@@ -25,7 +25,7 @@ AC_PREREQ([2.68])
 
 dnl Version number stuff here:
 
-AC_INIT([libpng],[1.6.38.git],[png-mng-implement@lists.sourceforge.net])
+AC_INIT([libpng],[1.6.38.magma],[png-mng-implement@lists.sourceforge.net])
 AC_CONFIG_MACRO_DIR([scripts])
 
 # libpng does not follow GNU file name conventions (hence 'foreign')
@@ -46,7 +46,7 @@ dnl automake, so the following is not necessary (and is not defined anyway):
 dnl AM_PREREQ([1.11.2])
 dnl stop configure from automagically running automake
 
-PNGLIB_VERSION=1.6.38.git
+PNGLIB_VERSION=1.6.38.magma
 PNGLIB_MAJOR=1
 PNGLIB_MINOR=6
 PNGLIB_RELEASE=38
diff --git a/codebase/libpng16/scripts/pnglibconf.dfa b/codebase/libpng16/scripts/pnglibconf.dfa
index 4dcaf7c..ea8476c 100644
--- a/codebase/libpng16/scripts/pnglibconf.dfa
+++ b/codebase/libpng16/scripts/pnglibconf.dfa
@@ -328,7 +328,7 @@ option WRITE_INT_FUNCTIONS disabled
 #     because it means that a failure to write an ancillary chunk can often be
 #     ignored.
 
-option WARNINGS
+option WARNINGS disabled
 option ERROR_TEXT
 option ERROR_NUMBERS disabled
 
@@ -362,7 +362,7 @@ option FIXED_POINT
 # macros and functions (FILE *, printf, etc.) from being compiled and used,
 # unless (PNG_DEBUG > 0) has been #defined.
 
-option STDIO
+option STDIO disabled
 option CONSOLE_IO requires STDIO
 
 # Note: prior to 1.5.0 this option could not be disabled if STDIO
@@ -408,8 +408,8 @@ option USER_LIMITS requires READ
 # Only chunks that are variable in number are counted towards the
 
 # Use 0x7fffffff for unlimited
-setting USER_WIDTH_MAX default        1000000
-setting USER_HEIGHT_MAX default       1000000
+setting USER_WIDTH_MAX default        0x7fffffff
+setting USER_HEIGHT_MAX default       0x7fffffff
 
 # Use 0 for unlimited
 setting USER_CHUNK_CACHE_MAX default     1000
@@ -518,7 +518,7 @@ option BUILD_GRAYSCALE_PALETTE
 
 # WRITE options
 
-option WRITE enables WRITE_INT_FUNCTIONS
+option WRITE disabled
 
 # Disabling WRITE_16BIT prevents 16-bit PNG files from being
 # generated.

